### PR TITLE
Add examples of custom keys

### PR DESCRIPTION
**Pull request by jakeouellette**
Add a variety of common custom keys stored on Android that can help developers better diagnose their crashes.

### CODE REVIEW

File: crash/app/src/main/java/com/google/samples/quickstart/crash/kotlin/CustomKeySamples.kt [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+     */
+    fun setSampleCustomKeys() {
+        FirebaseCrashlytics.getInstance().setCustomKeys(CustomKeysAndValues.Builder()
+                .putString("Locale", locale)
+                .putFloat("Screen Density", density)
+                .putString("Google Play Services Availability", googlePlayServicesAvailability)
+                .putString("Os Version", osVersion)
+                .putString("Install Source", installSource)
+                .putString("ABIs", preferredAbi).build())
+    }
```
Comment (rosariopfernandes) [inline crash/app/src/main/java/com/google/samples/quickstart/crash/kotlin/CustomKeySamples.kt:38]: Crashlytics KTX already had a [setCustomKeys](https://firebase.google.com/docs/reference/kotlin/com/google/firebase/crashlytics/ktx/package-summary#(com.google.firebase.crashlytics.FirebaseCrashlytics).setCustomKeys(kotlin.Function1)) method before the Java one was introduced.

Should we demonstrate that one instead? Or is the new one the most recommended?

File: crash/app/src/main/java/com/google/samples/quickstart/crash/kotlin/CustomKeySamples.kt [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                override fun onCapabilitiesChanged(network: Network, networkCapabilities: NetworkCapabilities) {
+                    updateNetworkCapabilityCustomKeys(networkCapabilities)
+                }
+            })
+        }
+    }
+
+    private fun updateNetworkCapabilityCustomKeys(networkCapabilities: NetworkCapabilities) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            val instance = FirebaseCrashlytics.getInstance()
```
Comment (rosariopfernandes) [inline crash/app/src/main/java/com/google/samples/quickstart/crash/kotlin/CustomKeySamples.kt:61]: For the Kotlin samples, we prefer to use Firebase KTX 
```suggestion
            val instance = Firebase.crashlytics
```

This applies throughout.

**Review by samtstern — APPROVED** [review]
Approved with Rosario's comments.

File: crash/app/src/main/java/com/google/samples/quickstart/crash/kotlin/CustomKeySamples.kt (hunk 1 above)
Comment (jakeouellette) [reply crash/app/src/main/java/com/google/samples/quickstart/crash/kotlin/CustomKeySamples.kt]: Great call out! We absolutely should. I probably should have used kotlin in the talk, retrospectively, as well... Alas.

File: crash/app/src/main/java/com/google/samples/quickstart/crash/kotlin/CustomKeySamples.kt (hunk 2 above)
Comment (jakeouellette) [reply crash/app/src/main/java/com/google/samples/quickstart/crash/kotlin/CustomKeySamples.kt]: updated

File: crash/app/src/main/java/com/google/samples/quickstart/crash/kotlin/CustomKeySamples.kt (hunk 1 above)
Comment (jakeouellette) [reply crash/app/src/main/java/com/google/samples/quickstart/crash/kotlin/CustomKeySamples.kt]: (updated)

**Comment by jakeouellette** [comment]
Rosario: Great callout! I've updated the code based on your suggestions.

**Review by rosariopfernandes — APPROVED** [review]

**Pull request merged**